integrations/access/pagerduty/client.go

```diff
@@ -79,27 +80,65 @@ func NewPagerdutyClient(conf PagerdutyConfig, clusterName, webProxyAddr string)
 		}
 	}
 
+	// APIEndpoint parameter is set only in tests
+	endPointURL := "https://api.pagerduty.com"
```

Comment:
Nit: Other plugins I think set the default API address as a top-level constant. Maybe move this one out as well for consistency?